checkov/terraform/checks/resource/aws/ALBListenerHTTPS.py

```diff
@@ -23,6 +23,12 @@ def scan_resource_conf(self, conf):
                 conf[key] == ["HTTPS"]
                 or
                 conf[key] == ["TLS"]
+                or
+                conf[key] == ["TCP"]
+                or
+                conf[key] == ["UDP"]
+                or
+                conf[key] == ["TCP_UDP"]
```

Comment:
@jdheyburn Might be more readable to replace this prolonged condition as it is now to:
`conf[key] in ("HTTPS", "TLS", "TCP"...)`

Reply:
Fair suggestion! I've made the change now 👍